### DISCUSSION TITLE
Create fabric.controlsUtils.changeHeight

### DISCUSSION THREAD

**Discussion by salomonsanz** [fabricjs/fabric.js#7377] [Ideas]
_fabric.controlsUtils.changeWidth_ exists, but _fabric.controlsUtils.changeHeight_ does not.

It is currently only used in default_controls.js to "override" ml and mr controls in TextBox

```
    textBoxControls.ml = new fabric.Control({
      x: -0.5,
      y: 0,
      actionHandler: controlsUtils.changeWidth,
      cursorStyleHandler: scaleSkewStyleHandler,
      actionName: 'resizing',
    });
```
It would be useful to reproduce the same behaviour with languages that are read vertically.
Also, to create easily "crop" controls, or resize shapes without skew or scale:

fabric.Image.prototype.controls.mt.actionHandler = fabric.controlsUtils.changeHeight; 
fabric.Image.prototype.controls.mb.actionHandler = fabric.controlsUtils.changeHeight; 
fabric.Rect.prototype.controls.mt.actionHandler = etc...

I will try to make a PR with this

**Comment by asturur** [comment]
We have it in the issue tracker somewhere.
You can open a PR with it, but you have to consider that until fabricJS is tree shakeable, that is just unused code. With support for vertical text this could change yes,
